packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -4663,55 +3552,20 @@ describe('NetworkController', () => {
   });
 
   describe('rollbackToPreviousProvider', () => {
-    /**
-     * The set of networks that, when specified, create an Infura provider as
-     * opposed to a "standard" provider (one suited for a custom RPC endpoint).
-     */
-    const INFURA_NETWORKS = {
```

Comment:
The extension already has this constant defined at the top of the file, and we need it in other tests, so I decided to move it.